Foundation/Host.swift

```diff
@@ -28,6 +28,10 @@ open class Host: NSObject {
     internal var _names = [String]()
     internal var _addresses = [String]()
     
+    #if os(Android)
```

Comment:
Nit: `#if` and `#endif` should be de-indented (no spaces at the beginning of the line before `#`) and the enclosed statements should be indented as though it isn't surrounded by a compile-time condition (that is, `static internal let NI_MAXHOST` here should align with `static internal let _current`).